doc/rose-configuration-metadata.html

```diff
@@ -831,6 +831,31 @@ <h3 id="meta.conf.behaviour">Metadata for Behaviour</h3>
 petits_fours    .false.     .true.       .false.       .true.
 </pre>
       </dd>
+      <dt id="meta.conf.behaviour.copy-mode">copy-mode (metadata useage with
+      the rose-suite.info file)</dt>
+
+      <dd>
+        <p>Setting copy-mode in the metadata allows for the field to be
+           either "never" copied or copied with any value associated 
+           to be "clear".</p>
```

Comment:
Please consider wrapping "never" and "clear" with `<code>...</code>` instead of quotes.